# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/socket/hotfix.patch
# Copyright (C) 2004 - 2025 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- socket-1.5p4/Makefile.vanilla	2025-07-15 19:36:51.211812643 +0200
+++ socket-1.5p4/Makefile	2025-07-15 19:37:18.594233316 +0200
@@ -32,7 +32,7 @@
 ### socket.1 in /usr/local/man/man1/socket.1
 ### Make sure the target directories exist before doing a "make install".
 
-INSTALLBASE = /usr/local
+INSTALLBASE = $(prefix)
 INSTALLBINPATH = $(INSTALLBASE)/bin
 INSTALLBINMODE = 755
 INSTALLMANPATH = $(INSTALLBASE)/man
